spec/system/planning_applications/updated_tab_spec.rb

```diff
@@ -79,6 +79,31 @@
     # Now perform an update to a planning application
     visit edit_planning_application_path(planning_application2)
 
+    within(find(:fieldset, text: "Agent information")) do
+      fill_in("Email address", with: "new_agent_email@example.com")
+    end
+
+    click_button("Save")
+    visit(root_path)
+    click_link "Updated"
+
+    last_audit = Audit.last
+    within("#audit_#{last_audit.id}") do
+      within(".govuk-inset-text") do
+        expect(page).to have_content(
```

Comment:
This part of the test will fail without the `next if saved_change_to_attribute(attribute_name).all? { |value| value.blank? || value.try(:zero?) }` changes

Reply:
It will expect the audit entry `"payment amount changed from £0.00 to £0.00"` instead which is what this PR is trying to get eliminate